presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java

```diff
@@ -655,6 +660,12 @@ public void setRoutines(List<RoutineInfo> routines)
         this.routines.set(ImmutableList.copyOf(routines));
     }
 
+    public void setDynamicFilterStats(DynamicFiltersStats dynamicFilterStats)
```

Comment:
For the consistent naming convention, the variable name can be `dynamicFiltersStats`. 
```suggestion
    public void setDynamicFilterStats(DynamicFiltersStats dynamicFiltersStats)
```

Reply:
done